bindings/node.js/Makefile

```
@@ -61,7 +61,7 @@ test: install
 # Lint js/ts code
 .PHONY: format
 format: install
-	@$(YARN) prettier --loglevel=warn --write .
+	@yarn eslint --quiet --color --ext .ts lib/ test/
```

I'd like for this to use the `$(YARN)` command to hide extraneous outputs.
```suggestion
	@$(YARN) eslint --quiet --color --ext .ts lib/ test/
```

Without it, I get this, which I don't find all that useful:
```
yarn run v1.22.19
$ /Users/jtraglia/Projects/mattk/c-kzg-4844/bindings/node.js/node_modules/.bin/eslint --quiet --color --ext .ts lib/ test/
```